pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -66,8 +66,17 @@ static __always_inline void read_into_buffer_skb(char *buffer, struct __sk_buff*
         *(&buffer[0]) = load_byte(skb, offset);
         asm volatile("":::"r1");
     }
+
+    return HTTP_BUFFER_SIZE <= (skb->len - (u32)offset) ? HTTP_BUFFER_SIZE - 1 : skb->len - (u32)offset;
 }
 
+SEC("socket/http_filter_stub")
```

Comment:
Can we name this `socket/http_filter_entry`? I think that might clarify the intent.